src/js/jsx/sections/transform/Position.jsx

```diff
@@ -53,7 +53,12 @@ define(function (require, exports, module) {
         },
 
         shouldComponentUpdate: function (nextProps, nextState) {
-            return this.state !== nextState;
+            return !this.state ||
```

Comment:
I'll give getInitialState a try, when I used it before, it was leaving the component unrendered, but that might have been the side effect of using `===`

`scrubX/scrubY` are states that don't affect rendering, and everytime we scrub, component receives new props to re-render. So it is intentional.